salt/zeek/soc_zeek.yaml

```diff
@@ -1,6 +1,6 @@
 zeek:
   enabled:
-    description: You can enable or disable ZEEK on all sensors or a single sensor.
+    description: Controls whether the Zeek (network packet inspection) process runs. Disabling this process could result in missed alerts and other important NIDS-related information. If Suricata was selected as the packet meta-data engine during setup then this will already be disabled.
```

Comment:
Controls whether the Zeek (network packet inspection) process runs. Disabling this process could result in loss of network protocol metadata. If Suricata was selected as the protocol metadata engine during setup then this will already be disabled.